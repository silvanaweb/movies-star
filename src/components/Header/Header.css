.Header {
  background-color: var(--teal);
  color: var(--teal-ligther);
  padding: var(--m-size);
}
