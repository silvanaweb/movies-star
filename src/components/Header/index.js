import React from "react";
import { connect } from "react-redux";
import "./style.css";

const Header = ({ history }) => {
  return (
    <header className="Header">
      <div className="content-container">
        <div className="Header__content">header</div>
      </div>
    </header>
  );
};

const mapStateToProps = (state, props) => {
  return {};
};

export { Header, categories };

export default connect(mapStateToProps)(Header);
