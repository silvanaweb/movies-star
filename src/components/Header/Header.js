import React from "react";
import "./Header.css";

const Header = () => (
  <header className="Header">
    <div className="content-container">
      <h1>Movies Stars</h1>
    </div>
  </header>
);

export { Header };
