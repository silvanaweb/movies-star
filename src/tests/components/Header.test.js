import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import { Header } from "../../components/Header/Header";

test("should render Header correctly", () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<Header />);
  expect(result).toMatchSnapshot();
});
