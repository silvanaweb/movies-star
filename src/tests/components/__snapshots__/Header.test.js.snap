// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header
  className="Header"
>
  <div
    className="content-container"
  >
    <h1>
      Movie Stars
    </h1>
  </div>
</header>
`;
